### PR TITLE
Fix/broken image table thumbnail

### PR DESCRIPTION
**Pull request by fti-smaske**
Description
This PR is for No Image in thumbnail for table to avoid broken image and remove unnecessary console logs.

In case image has url but its not valid then it will show No Image in thumbnail.

Related Bugherd Requirement links
[Requirement](https://www.bugherd.com/projects/216134/tasks/56)

Considerations on Implementation
Tested this work in TCC project.

image

### CODE REVIEW

**Review by fti-skumar — APPROVED** [review]
LGTM

File: src/Tables/atoms/TableRowThumbnail.tsx [hunk 1]
```diff
@@ -136,17 +149,44 @@ const TableRowThumbnail: React.FC<IProps> = ({ hoverZoom = true, image='', media
...
 
-
-
   const onLoad = useCallback(()=>{
     timeoutRef.current && clearTimeout(timeoutRef.current);
     timeoutRef.current = null;
     setShowImage(true);
   },[]);
 
+  const checkIfImageExists = (url: string, callback: (exists: boolean) => void) => {
```
Comment (fti-bcarpenter) [inline src/Tables/atoms/TableRowThumbnail.tsx:158]: callback is not a good name here. Use some different name like imageExistsCallback

**Review by fti-bcarpenter — CHANGES_REQUESTED** [review]

File: src/Tables/atoms/TableRowThumbnail.tsx (hunk 1 above)
Comment (fti-smaske) [reply src/Tables/atoms/TableRowThumbnail.tsx]: Done

**Review by fti-bcarpenter — APPROVED** [review]

**Review by atomworks — APPROVED** [review]
There was one thing I wasn't sure about so I've highlighted that in a comment for Josh to check. Other than that though, happy to approve.

**Pull request merged**